content/docs/self-hosted-runners.md

```diff
@@ -175,28 +175,20 @@ Click below to see credentials needed for supported cloud service providers.
 
 ### AWS
 
-```yaml
-env:
```

Comment:
Minor and unrelated to this PR but I noticed all these headers are H3 but should be H3 since their apparent parent it an H3 itself (`### Cloud credentials` in this case)

Reply:
\* should be H4

Reply:
> _are H3 but should be H3_

I assume you wante to say that «[headers] are H3 but should be H4» (?)

Reply:
I didn't pay it too much attention because they were `<details>`-like elements that get rendered to `<span>` anyway, but yes, they should proabably be H4 instead.

Reply:
@jorgeorpinel, would you prefer to fix this in a separate pull request?

Reply:
Yes a separate PR seems appropriate.

Reply:
Thanks for spotting this!

Reply:
Np. BTW do you need me to approve the PR? I assume the changes are correct but Idk 😁

Reply:
I need somebody to approve this pull request so it can be merged, but I'll leave this inside the CML team to avoid making you liable for my nitpicks. 😄  Thank you anyways!